### PR TITLE
Paw 152   Change stiles for principal pets home page

### PR DESCRIPTION
**Pull request by LauRisueno**
## CHECKLIST
**Sigue el checklist para ayudar a que tu PR se revice mas facil. Marca los puntos que ya completaste.**

- [x] Revisaste tu propio PR?

- [x] Agregaste una BREVE descripcion de este PR abajo.

- [x] Agregaste una descripcion de la SOLUCION de este PR abajo.

- [ ] Los cambios se aprobaron por diseño?

- [ ] Este PR paso QA?

## SUMMARY

In this ticket, we change the home page styles based on the Figma design. 

## SOLUTION

To resolve this ticket, we change both the HTML and CSS styles of the following components: adoption, filters, pet-tile, and pet-list.

## SCREENSHOTS

Captura de Pantalla 2023-04-14 a la(s) 18 40 35

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1,30 +1,10 @@
 @import "../node_modules/bootstrap/scss/functions";
 @import "../node_modules/bootstrap/scss/variables";
-@import "~bootstrap-icons/font/bootstrap-icons.css";
-@import "~@ng-select/ng-select/themes/default.theme.css";
```
Comment (erwinXZ) [inline src/styles.scss:4]: Why are these 2 imports removed?

Comment (LauRisueno) [reply src/styles.scss:4]: Sorry I didn't realize that, I'll correct it right now

**Review by erwinXZ — COMMENTED** [review]
The "Detalle" button does not exists but the card must redirect to the pet detail

**Comment by LauRisueno** [comment]
> The "Detalle" button does not exists but the card must redirect to the pet detail

I updated the pet-tile component with a routerLink that redirects to pet details.

**Review by erwinXZ — APPROVED** [review]

**Review by erwinXZ — APPROVED** [review]

**Pull request merged**